app/src/main/res/xml/preferences.xml

```diff
@@ -60,11 +60,6 @@ along with SwiFTP.  If not, see <http://www.gnu.org/licenses/>.
                 android:entryValues="@array/pref_list_themes_values"
                 android:key="theme"
                 android:title="@string/app_theme" />
-            <CheckBoxPreference
-                android:defaultValue="true"
-                android:key="show_notification_icon_preference"
-                android:summary="@string/show_notification_icon_summary"
-                android:title="@string/show_notification_icon" />
```

Comment:
It'd be better if this preference is still available.

Reply:
If you do startForegroundService the notification is mandatory cannot be dismissed by the user. Yes you are right maybe can be replaced by contextcompact.

Reply:
@constambeys The preference is for hiding the small icon on the status bar.

Reply:
Are you sure it works because the ACTION_UPDATE_NOTIFICATION is not registered in the intentFilter

     val showNotificationIconPref = findPref("show_notification_icon_preference");
        showNotificationIconPref.setOnPreferenceChangeListener((preference, newValue) -> {
            getActivity().sendBroadcast(new Intent(FsNotification.ACTION_UPDATE_NOTIFICATION));
            return true;
        });

Reply:
i am not sure why that is the way it is. Regardless, the preference would work fine if FTP server is started after toggling it.
https://github.com/ppareit/swiftp/blob/43440fe44aebe158b595ef97c2a1238ccf96a262/app/src/main/java/be/ppareit/swiftp/gui/FsNotification.java#L51-L55

Reply:
I attached a debugger and this code is never called. Try it yourself.I see that this button is to update the notification but I don't know if this can be adjusted with the new code changes

Reply:
I know its never called. What I am saying is that it is there for updating the notification in case the FTP Server was already running. The preference would work fine regardless of the receiver because the code to toggle small icon on the notification lives inside `setupNotification()`. This comes with the obvious limitation of not being able to update the notification on running server.

Reply:
Probably but in my opinion is not a major issue. If you toggle the server the notification will be updated.

Reply:
Exactly what I am saying. Create a new commit to include suggested changes and keep this preference as is.

Reply:
ok

Reply:
> Are you sure it works because the ACTION_UPDATE_NOTIFICATION is not registered in the intentFilter
> 
> ```
>  val showNotificationIconPref = findPref("show_notification_icon_preference");
>     showNotificationIconPref.setOnPreferenceChangeListener((preference, newValue) -> {
>         getActivity().sendBroadcast(new Intent(FsNotification.ACTION_UPDATE_NOTIFICATION));
>         return true;
>     });
> ```

Since versions O of the android API, some broadcasts are no longer send. In a previous commit (f53c2ca4e5be13fb9d1fbbee67d88dbecbde283f) I changed "all" impliciet broadcasts. This is probably a broadcast that still has to be fixed.

WIth the suggested changes from @ashutoshgngwr we can pull in your commit.